packages/discovery-react-components/src/components/DocumentPreview/DocumentPreview.tsx

```diff
@@ -4,26 +4,28 @@ import { settings } from 'carbon-components';
 import { QueryResult, QueryResultPassage, QueryTableResult } from 'ibm-watson/discovery/v2';
 import { SearchContext } from 'components/DiscoverySearch/DiscoverySearch';
 import { PreviewToolbar } from './components/PreviewToolbar/PreviewToolbar';
+import PdfViewer from './components/PdfViewer/PdfViewer';
 import SimpleDocument from './components/SimpleDocument/SimpleDocument';
 import withErrorBoundary, { WithErrorBoundaryProps } from 'utils/hoc/withErrorBoundary';
 import { defaultMessages, Messages } from './messages';
 import HtmlView from './components/HtmlView/HtmlView';
-import { isCsvFile, isJsonFile } from './utils/documentData';
+import { getTextMappings, isCsvFile, isJsonFile } from './utils/documentData';
+import { TextMappings } from './types';
 
 const { ZOOM_IN, ZOOM_OUT } = PreviewToolbar;
 
 interface Props extends WithErrorBoundaryProps {
   /**
-   * Document data, as that returned by a query. Overrides result from SearchContext
+   * Document data, as that returned by a query. Overrides result from SearchContext.
    */
   document?: QueryResult;
   /**
-   * PDF file data as base64-encoded string
+   * PDF file data as "binary" string (array buffer). Overrides result from SearchContext.documentProvider.
```

Comment:
Turns out previous comment was false. This `file` variable goes straight into `getDocument()` from `pdfjs-dist`, which takes an array buffer (not base64 encoded).